api/http/handler/stacks/handler.go

```diff
@@ -18,9 +18,10 @@ import (
 	"github.com/portainer/portainer/api/internal/authorization"
 	"github.com/portainer/portainer/api/scheduler"
 	"github.com/portainer/portainer/api/stacks"
+	"github.com/sirupsen/logrus"
 )
 
-const defaultGitReferenceName = "refs/heads/master"
```

Comment:
if this is a sane fix, then I presume this variable should just go away.

Reply:
it that works, then it's a sane fix 😝 And if so would be nice to remove the field and payload validators that use it.